lib/openstudio-standards/standards/necb/NECB2011/necb_2011.rb

```diff
@@ -218,6 +218,7 @@ def get_necb_hdd18(model:, necb_hdd: true)
   # This method is a wrapper to create the 16 archetypes easily.
   def model_create_prototype_model(template:,
                                    building_type:,
+                                   construction_opt:'',
```

Comment:
New option: If set to 'structure', default construction sets are based on underlying building _structure_ (vs previous BTAP approach).